content/cli/ingest/ingest-examples.md

```diff
@@ -225,8 +225,11 @@ The specified container image is expected to conform to the following interface:
 - Use following environment variables:
   - `ODF_LAST_MODIFIED` - last modified time of data from the previous ingest run, if any (in RFC3339 format)
   - `ODF_ETAG` - caching tag of data from the previous ingest run, if any
+  - `ODF_BATCH_SIZE` - batch size for the current ingest run; default value is 10 000, can be overrided via `env`
```

Comment:
I think we should provide a more expanded explanation. Tell that this is the recommended number of records, that it's only for scripts that provide continuous stream of data and can resume from previous state.

Reply:
@sergiimk , yes, it's a good point to highlight the recommendation aspect -- updated